modules/azurerm/Firewall-NAT-Rules/variables.tf

```diff
@@ -0,0 +1,47 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+#
+# This software is the property of WSO2 LLC. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+# You may not alter or remove any copyright or other notice from copies of this content.
+#
+# --------------------------------------------------------------------------------------
+
+variable "resource_group_name" {
+  description = "Resource group name"
+  type        = string
+}
+
+variable "firewall_name" {
+  description = "Firewall name"
+  type        = string
+}
+
+variable "firewall_nat_rule_collection_name" {
+  description = "Firewall nat rule collection name"
+  type        = string
+}
+
+variable "priority" {
+  description = "Priority"
+  type        = number
+}
+
+variable "action" {
```

Comment:
Let's remove this and hardcode the action as the module is is only intended for DNAT